runtime/polkadot/src/lib.rs

```diff
@@ -110,7 +110,12 @@ pub struct BaseFilter;
 impl Filter<Call> for BaseFilter {
 	fn filter(call: &Call) -> bool {
 		match call {
-			Call::Parachains(parachains::Call::set_heads(..)) => true,
+			Call::Parachains(parachains::Call::set_heads(..))
+				| Call::Democracy(democracy::Call::vote(..))
```

Comment:
Maybe we should move the allowed calls from the bottom to the top here as well?

Reply:
It wouldn't gain much as there's usage of defaults for selection happening both ways (e.g. `Call::Vesting(vesting::Call::vested_transfer(..)) | Call::Indices(indices::Call::transfer(..))`)